SU-SWS/decanter
# Clean up, more components and elements

# READY FOR REVIEW

# Summary
- Added several components, elements, and core items (tickets in project represent what is in progress; also cleaned up duplicated or deprecated tickets in project)
- Cleaned up all KSS and SCSS comments and code spacing for uniformity
- Ensured modifier classes being used in Twig templates were generating a good-looking default style (e.g., alerts and buttons)
- Began documenting all items (some more thoroughly than others)
- Organized components at proper levels in file system
- Added necessary files for all components, elements, mixins, etc. to be added to style guide (with placeholder text/markup where needed)
- Added overflow mixin to clean up markup being generated

# Needed By (Date)
- N/A

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that things are looking good and much more flushed out in local pattern library and style guide
3. Cleaner code base

# Affected Projects or Products
- Decanter

# Associated Issues and/or People
- https://github.com/SU-SWS/decanter/projects/1

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
